schema_editor/test/spec/views/recordtype/add-controller.spec.js

```diff
@@ -1,115 +0,0 @@
-'use strict';
```

Comment:
What led you to remove these tests completely? I see from the commit history that you had added them at one point and it looks like you were trying to get at least some of them working.

Reply:
Porting the tests over from the schema editor worked fine, but for some reason when I tried to move tests for the records service from the `web` app into the schema editor, nearly all the tests were failing for totally incomprehensible reasons. (e.g. the built-in mocked `$.httpBackend` fixture was undefined for most of the tests, which as far as I could glean was a side-effect of other directives not being imported properly?) I made almost no progress in an hour or so of debugging so decided that it wasn't worth it.

I'd be happy to pair on debugging the test failures! I just assumed that since I planned on rewriting a good portion of this app anyway, it wasn't necessarily a good use of our time.

Reply:
Cool, nope, that sounds like exactly the right thing to do, just wanted to hear a bit of the back story in case there was something I could easily help with, but that doesn't sound like an easy problem and I agree that it's not worth spending a ton of time on this.